doc/source/contributor/environments/aufn-ceph.rst

```diff
@@ -181,9 +188,8 @@ We can deploy a test VM to ensure that our 'universe' is up and running by first
 
 .. parsed-literal::
 
-    VENV_DIR=~/openstack-env
-    python -m venv $VENV_DIR
-    source $VENV_DIR/bin/activate
+    python -m venv ~/openstack-env
```

Comment:
Does this need to be `python3`? CentOS/Rocky no longer have an unversioned system `python`

Reply:
Yes that does :+1: